src/amo/css/Home.scss

```diff
@@ -14,45 +14,34 @@
   overflow: auto;
   padding: 0;
   width: 100%;
+}
 
-  li {
-    background-color: #fff;
-    background-size: 50% 50%;
-    border-radius: 6px;
-    display: block;
-    flex-grow: 1;
-    list-style-type: none;
-    margin: 5px;
-    overflow: auto;
-    padding: 0;
-    position: relative;
-    text-align: center;
-    width: 30%;
-
-    &::before {
-      content: '';
-      display: block;
-      padding-bottom: 100%;
-    }
-  }
+.HomePage-cat-li {
```

Comment:
I know the extra characters make the HTML longer but can we make this `.HomePage-category-li`? The abbreviations always trip me up and I'd like to discourage their use 😄